### PR TITLE
Widget disposed check

### PR DESCRIPTION
**Pull request by theJNOVAK**
issue: #1566

### CODE REVIEW

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/WidgetLookup.java [hunk 1]
```diff
@@ -429,6 +429,9 @@ public Control run() {
 	 * @return true if widget is visible, false otherwise
 	 */
 	private boolean visible(final Widget w) {
+		if (w.isDisposed())
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/WidgetLookup.java]: Use braces even if condition is one line

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/WidgetHandler.java [hunk 2]
```diff
@@ -104,6 +104,9 @@ public Boolean run() {
 	 * @return true if widget is visible, false otherwise
 	 */
 	public boolean isVisible(Widget widget) {
+		if (widget.isDisposed())
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/WidgetHandler.java]: Roll back those changes in handler. We do not check whether a widget is disposed in handler methods.

**Comment by mlabuda** [comment]
Next time please use hub to transform an existing issue into a new PR.

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/WidgetLookup.java (hunk 1 above)
Comment (theJNOVAK) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/WidgetLookup.java]: edited

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/WidgetHandler.java (hunk 2 above)
Comment (theJNOVAK) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/WidgetHandler.java]: rolled back

**Pull request merged**